templates/cloud/openstack/index.html

```diff
@@ -19,145 +19,87 @@
 
         </div>
         <div class="four-col last-col">
-            {% comment %} might want to add this image instead of the chart - {{ ASSET_SERVER_URL }}845e046c-partner-openstack.png
-            {% endcomment %}
-            <figure id="ubuntu-pie" class="ubuntu-pie"></figure>
-            <p class="note">Among OpenStack deployments with more than 1,600 users,
-              <a href="https://www.openstack.org/assets/survey/April-2016-User-Survey-Report.pdf" class="external">OpenStack survey &mdash; April 2016 [PDF 14.6MB]</a></p>
+            {% include "cloud/shared/_openstack-pie-chart.html" %}
         </div>
     </div>
 </section>
 
-<script src="{{ ASSET_SERVER_URL }}3c99518b-d3-version3.5.6.min.js"></script>
-
-<script>
-  'use strict';
-  var width = document.querySelector('#ubuntu-pie').offsetWidth;
-
-  var dataset = [
-    { label: 'ubuntu', count: 55, left: (width / 2.75), top: -5 },
-    { label: 'other', count: 45, left: -(width / 2.75), top: -5 }
-  ];
-  var height = width;
-  var radius = Math.min(width, height) / 2;
-  var donutWidth = width / 4;
-  var color = d3.scale.category20b();
-  var svg = d3.select('#ubuntu-pie')
-    .append('svg')
-    .attr('width', width)
-    .attr('height', height)
-    .attr('viewBox', '0 0 ' + (width + 2) + ' ' + (height + 2))
-    .append('g')
-    .attr('transform', 'translate(' + (width / 2 + 1) +
-      ',' + (height / 2 + 1) + ')');
-
-  var arc = d3.svg.arc()
-    .innerRadius(radius - donutWidth)
-    .outerRadius(radius);
-
-  var pie = d3.layout.pie()
-    .value(function(d) { return d.count; })
-    .sort(null);
-
-  svg.selectAll('path')
-    .data(pie(dataset))
-    .enter()
-    .append('path')
-    .style('stroke', '#aea79f')
-    .style('stroke-width', '1')
-    .style("stroke-dasharray", ("3, 3"))
-    .attr('d', arc)
-    .attr('class', function(d, i) {
-      return d.data.label;
-    });
-
-  // Add labels as <text> elements
-  var labels = svg.selectAll('foreignObject')
-    .data(dataset)
-    .enter()
-    .append('text')
-    .attr('y', function(d) { return d.top; })
-    .attr('class', function(d) { return d.label + '-label'; });
-
-  // Fill <text> labels with <tspan>s
-  labels.append('tspan')
-    .text(function (d) {return d.label;})
-    .attr('x', function(d) {
-      var spanWidth = 43;
-      if( d.label === 'ubuntu') { spanWidth = 53 }
-      return d.left - spanWidth / 2;
-    })
-    .attr('dy', '0');
-  labels.append('tspan')
-    .text(function (d) {return '(' + d.count + '%)'})
-    .attr('x', function(d) { return d.left - 44 / 2; })
-    .attr('dy', '20');
-</script>
+<section class="row strip-light">
+    <div class="strip-inner-wrapper">
+    {% include "cloud/shared/_openstack_clients.html" %}
```

Comment:
I think this row should be below the "Four ways" row, or at least that's how it is in the copydoc.